HunterPie.Core/Game/Rise/Entities/MHRMonster.cs

```diff
@@ -268,13 +268,26 @@ private void GetMonsterHealthData()
     [ScannableMethod]
     private void GetMonsterCaptureThreshold()
     {
+        if (MonsterType == MonsterType.Qurio)
+        {
+            CaptureThreshold = 0.0f;
+            return;
+        }
+
+        MonsterDataSchema? data = MonsterData.GetMonsterData(Id);
+        if (data.HasValue && data.Value.Capture == 0)
```

Comment:
I think a better approach would be to have a new field in the xml, like `IsCapturable` or something along these lines. It's better than using `Capture` as a boolean.

Reply:
I was trying to not add or change something more. If I were to add an IsNotCapturable field, would it be a Bool value?

Reply:
Yes, either a `IsNotCapturable` with default value as `false` or a `IsCapturable` with default value as `true` in the parser.

Reply:
Sorry for taking time to work. Please review.